docs/guides/abn-tests.mdx

```diff
@@ -53,59 +48,56 @@ Finally, don't forget to click on the **Save** button at the top right hand side
 
 ![image](https://user-images.githubusercontent.com/1315028/138973362-69b53d48-745a-462f-a275-b0a3720d4d90.png)
 
-## Step 2: Install the Statsig SDK
+## Step 2: Install and Initialize the Statsig SDK
 
 In this guide, we use the Statsig Javascript client SDK. See the Statsig [client or server SDKs docs](https://docs.statsig.com/#sdks) for your desired programming language and technology.
 
 <Install />
 
-## Step 3: Initialize the SDK
-
 After you install the SDK, you must initialize the SDK using an active **Client API Key** from the **API Keys** tab under **[Settings](https://console.statsig.com/settings)** in the Statsig console.
 
 In addition to the Client API key, you must also pass in a `Statsig User ID` to ensure users can be assigned and tracked for different variants of the experiment.
 
 <Initialize />
 
-## Step 4: Check the new experiment in your application
+## Step 3: Check the experiment in your application and log an event
 
 Get your experiment's configuration parameters to enable the user to see the experience you want to create with each variant. In this case, we fetch the value of the shape parameter that we had created earlier.
 
 ```jsx
-const expConfig = statsig.getExperiment("product_logo_icon_shapes");
+const expConfig = statsig.getExperiment("onboarding_banner");
 
-const buttonShape = expConfig.get("shape", "rounded_square");
+const showBanner = expConfig.get("showbanner", false);
  
 ```
 
 Now when a user renders this page in their client application, you will automatically start to see a live log stream in the Statsig console when you navigate to your experiment and select the **Diagnostics** tab.
 
 ![image](https://user-images.githubusercontent.com/1315028/139113152-7d19afa1-733f-4ee4-b788-3d39cecedae1.png)
 
-## Step 5: Log an event
-
-After you have set up the experiment, you may want to track downstream events to measure how your users are responding to different variants of the experiment. To do this, you can call the `LogEvent` API with the key measure that you may want to aggregate as well as other metadata that you may use as additional dimensions to breakdown the aggregate results.
+At this stage, you may also want to track downstream events to measure how your users are responding to different variants of the experiment. To do this, call the `LogEvent` API with the key measure that you may want to aggregate as well as other metadata that you may use as additional dimensions to breakdown the aggregate results.
 
 ```jsx
 statsig.logEvent('add_to_cart', 'groceries', {'price': '9.99', 'SKU_ID': 'SKU-123'});
 ```
 
-## Step 6: Review experiment results
+As the experiment progresses, you will see how many experiment checks occur and how many events are logged on an hourly basis in the **Diagnostics** tab.
+
 
-As experiment progresses, you will see how many experiment checks have been performed on a daily basis in the **Diagnostics** tab.
+## Step 4: Review experiment results
```

Comment:
Should we touch on the diagnostics tab first, as that gives you some real time indication of the rollout, and then the 24 hour later results/data?